src/screens/About/About.tsx

```diff
@@ -87,6 +87,7 @@ const About = ({ children }: { children: React.ReactNode }) => {
                       image={logo.image}
                       url={logo.url}
                       altText={logo.altText}
+                      key={i}
```

Comment:
Consider using a different letter here since both `i` are in scope here (the inner `i` wins, but just to make it explicit)